oio/blob/mover.py

```diff
@@ -193,6 +194,18 @@ def chunk_move(self, path, chunk_id):
         content_id = meta['content_id']
         chunk_id = meta['chunk_id']
 
+        # Maybe skip the chunk because it doesn't match the size constaint
+        chunk_size = int(meta['chunk_size'])
+        min_chunk_size = int(self.conf.get('min_chunk_size', 0))
+        max_chunk_size = int(self.conf.get('max_chunk_size', 0))
```

Comment:
We could initialize attributes in the constructor for these values.

Reply:
This is intended to be a minimal change. If a wrong configuration value is set (e.g. a string), we behave *much* better like this because the exception is cleanly catched and the error is cleanly reported. If the value is parsed in the constructor, we behave much worse (we loop and fill the logs with ugly backtraces).